gapii/cc/spy.cpp

```diff
@@ -459,13 +501,19 @@ void Spy::observeFramebuffer(uint8_t api) {
     if (downsamplePixels(data, w, h,
                          &downsampledData, &downsampledW, &downsampledH,
                          kMaxFramebufferObservationWidth, kMaxFramebufferObservationHeight)) {
-        atom_pb::FramebufferObservation observation;
-        observation.set_originalwidth(w);
-        observation.set_originalheight(h);
-        observation.set_datawidth(downsampledW);
-        observation.set_dataheight(downsampledH);
-        observation.set_data(downsampledData.data(), downsampledData.size());
-        mEncoder->message(&observation);
+        // atom_pb::FramebufferObservation observation;
+        auto observation = std::unique_ptr<atom_pb::FramebufferObservation>(
+            new atom_pb::FramebufferObservation());
```

Comment:
Hmm, we really don't have make_unique. Should we add one? In this CL or a separate one? @ben-clayton